src/components/Notification/Notification.styles.tsx

```diff
@@ -0,0 +1,54 @@
+import color from "../../styles/colors";
+import fonts from "../../styles/fonts";
+import resetCSS from "../../styles/reset";
+
+const initialStyles = `
+    ${resetCSS}
+    ${fonts.text}
+    background: #112F5C;
+    border-radius: 20px;
+    color: white;
+    min-width: 270px;
+    max-width: 25%;
+    height: 84px;
```

Comment:
A fixed height seems dangerous, have you tested this with a long string of text? I would remove this and use padding to push the text off the edges of the component if needed. This will mean you prob dont need to flex the component either

Reply:
I 've used % . Seems I changed it for whatever reason. Will correct that